src/core/packet_builder.c

```diff
@@ -197,6 +197,7 @@ QuicPacketBuilderPrepare(
         Connection->Stats.QuicVersion == QUIC_VERSION_2 ?
             QuicKeyTypeToPacketTypeV2(NewPacketKeyType) :
             QuicKeyTypeToPacketTypeV1(NewPacketKeyType);
+    BOOLEAN GreaseQuicBitTPReceived = (BOOLEAN)((Connection->PeerTransportParams.Flags & QUIC_TP_FLAG_GREASE_QUIC_BIT) > 0);
```

Comment:
```suggestion
    BOOLEAN NoGreaseQuicBit = !(Connection->PeerTransportParams.Flags & QUIC_TP_FLAG_GREASE_QUIC_BIT);
```
And then use `NoGreaseQuicBit` below?